### PR TITLE
trace: Add more filters options

### PR DESCRIPTION
**Pull request by vadmeste**
Filter options are necessary if the server receives a lot of requests,
especially when verbose mode is passed.

### CODE REVIEW

**Comment by harshavardhana** [comment]
@vadmeste can you add some examples for this in the `mc admin trace --help` ?

**Review by harshavardhana — APPROVED** [review]
This is brilliant work! :cake:

File: cmd/admin-trace.go [hunk 1]
```diff
@@ -43,6 +45,18 @@ var adminTraceFlags = []cli.Flag{
 		Name:  "all, a",
 		Usage: "trace all traffic (including internode traffic between MinIO servers)",
 	},
+	cli.StringFlag{
+		Name:  "status-codes",
+		Usage: "trace only a specific status codes",
```
Comment (poornas) [inline cmd/admin-trace.go:50]:
```suggestion
		Usage: "trace only specific status codes",
```

**Review by poornas — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**